tutor/hooks/consts.py

```diff
@@ -146,15 +160,17 @@ def your_filter(items):
     #: :parameter str name: basename of the host-mounted folder. In the example above,
     #:   this is "edx-platform". When implementing this filter you should check this name to
     #:   conditionnally add mounts.
-    COMPOSE_MOUNTS = filters.get("compose:mounts")
+    COMPOSE_MOUNTS: "Filter[t.List[t.Tuple[str, str]], [str]]" = filters.get(
+        "compose:mounts"
+    )
```

Comment:
I think using NamedTuples here could really improve the readability of some of these definitions. It would be backwards-compatible, since NamedTuples are can be used as ordinary tuples.

```suggestion
    class ComposeMount(t.NamedTuple):
        service: str
        path: str

    COMPOSE_MOUNTS: Filter[list[ComposeMount], [str]] = filters.get(
        "compose:mounts"
    )
```

Could be a future PR, of course.

Reply:
Let's keep this for a future PR.